src/Microsoft.AspNet.Mvc.Razor/Internal/RouteUtility.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.AspNet.Mvc.Razor.Internal
+{
+    /// <summary>
+    /// Utility for normalizing route values.
+    /// </summary>
+    public class RouteUtility
+    {
+        /// <summary>
+        /// In a case sensitive file system, Views with a case different from the URL will not be found
```

Comment:
Perhaps this should belong in the `remarks` section? The summary should be more along the lines of 

> Gets the value for a route key from the route constraints or route value defaults of an `<see cref="ActionDescriptor"/>` if it exists there, otherwise from the `RouteData`. 

(or something along those lines)